src/routes/index.svelte

```diff
@@ -31,16 +31,26 @@
 
 	/* Components */
 	import GameCardHighlighted from '$lib/Cards/GameCardHighlighted.svelte';
-	import Carousel from '$lib/GameViews/Carousel.svelte';
+	import Carousel from '$lib/views/Carousel.svelte';
+	import MijnGymles from '$lib/views/mijnGymles.svelte';
 
 	export let games: Game[];
 	const newestGames = games.slice(Math.max(games.length - 4, 0));
 </script>
 
 <main class="leaves-bg">
-	<GameCardHighlighted />
-	<Carousel gamesData={newestGames} carouselTitle="Nieuwe spellen" />
-	<Carousel gamesData={games} carouselTitle="Favoriete spellen" />
+	<section>
+		<h1>Uitgelichte spellen</h1>
```

Comment:
Denk dat hier nog wel conflictje komt. Kijk maar even wat slim is als je die van mij gemerged hebt. Want volgens mij staat die bij mij nog in het component (GameCardHighlighted) zelf